text/0032-nft-standard.md

```diff
@@ -93,13 +130,50 @@ The AttributeObject has the following JSON structure:
 
 The AttributeObject may have more attributes than the ones described above but those are the required ones.
 
+- The RoyaltyObject has the following JSON structure:
+
+```
+{
+    "fee_basis_points": {
+        "type": "integer",
+        "description": "Royalties percentage awarded to creators (basis points), e.g. 500, which means 5%."
+    },
+    "creators": {
+        "type": "Array<CreatorObject>",
+        "description": "Array of all creators involved in the project & their royalty bps."
+    }
+}
+```
+
+500 Basis Points mean 5% royalty, One basis point is equal to 1/100th of 1%. More info on BPS at [Investopedia](https://www.investopedia.com/terms/b/basispoint.asp).
+
+- The CreatorObject has the following JSON structure:
+
+```
+{
+    "address": {
+        "type": "string",
+        "description": "Address of the wallet in which royalty fees will be sent for secondary sales."
+    },
+    "share": {
+        "type": "integer",
+        "description": "Amount of royalty for each creator in bps, totaling a sum of 10000."
```

Comment:
I think it's better to be more detailed when explaining the shares, so giving examples and more explanation below. Here I'd just say: "Amount of royalty for each creator, in bps"

Reply:
Done